### PR TITLE
Update Indonesian translation

### PR DESCRIPTION
**Pull request by triyanwn**
Current progress is 50% translated.

### CODE REVIEW

File: po/id.po [hunk 1]
```diff
@@ -9,8 +9,8 @@ msgstr ""
 "Project-Id-Version: GnuCash 4.1\n"
 "Report-Msgid-Bugs-To: https://bugs.gnucash.org/enter_bug.cgi?"
 "product=GnuCash&component=Translations\n"
-"POT-Creation-Date: 2020-09-02 10:44+0700\n"
-"PO-Revision-Date: 2020-09-05 11:36+0700\n"
+"POT-Creation-Date: 2020-09-15 20:18+0700\n"
+"PO-Revision-Date: 2020-09-17 20:40+0700\n"
 "Last-Translator: Triyan W. Nugroho <triyan.wn@gmail.com>\n"
 "Language-Team: Indonesian <translation-team-id@lists.sourceforge.net>\n"
```
[the remark this answers is not in the document]
Comment (fellen) [reply po/id.po:15]: I have fixed it in commit 5de41ab. Other changes there:
For some unknown reason in the glossary PROJECT was not replaced.
Our release manager has to inform the TP manager that you send your changes directly to us and not their id team. Until that is done, it belongs in NEW_LINGUAS in CMakeLists.txt

**Review by fellen — APPROVED** [review]
Nice work, thanks!

**Comment by fellen** [comment]
Because I have rewritten the commit message, it became commit b823431.
Merged!

File: po/id.po (hunk 1 above)
[the remark this answers is not in the document]
Comment (triyanwn) [reply po/id.po:15]: Thanks, I will check it for future work.

File: po/id.po [hunk 2]
```diff
@@ -31325,3 +31098,169 @@ msgstr "status transaksi: dibatalkan"
 #: libgnucash/tax/us/txf.scm:98
 msgid "No help available."
 msgstr "Tak ada bantuan yang tersedia."
+
+#~ msgid "Change a Business Association"
+#~ msgstr "Ubah Asosiasi Bisnis"
+
```
[the remark this answers is not in the document]
Comment (triyanwn) [reply po/id.po:31104]: Thanks again for the tip!

**Comment by fellen** [comment]
@triyanwn By running I18nspector I got the following error message:
E: id.po: unusual-plural-forms 'nplurals=2; plural=n != 1;' => 'nplurals=1; plural=0;'

That  would mean, by example in English I would write:
> He has 1 book
> He has 2 book**s**.

You see the difference?
 But in indonesian:
> Dia memiliki 1 buku.
> Dia memiliki 2 buku.

you can use the same form. If that is correct you should

1. adjust that line in the header,

1. drop everywhere the 2. occurence of msgstring

**Comment by triyanwn** [comment]
For some sentences there are no difference. But for example for these sentences:

> The book has been read.
> The books have been read.

I would write in Indonesia:

> Buku itu sudah dibaca.
> Buku-buku itu sudah dibaca.

According to [GNU Gettext Manual](https://www.gnu.org/software/gettext/manual/html_node/Plural-forms.html), Bahasa Indonesia should use `nplurals=2; plural=n != 1;`

**Comment by fellen** [comment]
Thanks, then there is an error in I18nspector's database.

**Pull request closed without merging**